### ISSUE TITLE
Clarity about chrome

### ISSUE THREAD

**Issue by 3n-mb** [diracdeltas/signal-muon#46]
It sounds like brave is using chromium. Electron is using chromium. Am I missing some difference between the two uses? Licensing?

**Comment by 3n-mb** [comment]
From main page of this repo:

> Signal Desktop built on top of Muon so you can run it without Google Chrome.

What's the deal?

**Comment by leppert** [comment]
@3n-mb I'm not sure I follow the question. Are you asking why the README says "you can run it without Google Chrome" when, in fact, Chromium underpins the project?

**Comment by diracdeltas** [comment]
> Am I missing some difference between the two uses? Licensing?

see https://github.com/brave/muon/blob/master/README.md. 

in addition, muon incorporates more chromium code, including the extensions system.

muon is also staying much closer to the chromium release schedule, which allows us to ship security updates faster.

**Comment by 3n-mb** [comment]
@leppert yes, since electron also [claims](https://electron.atom.io/docs/tutorial/about/) to be based on chromium.

**Comment by 3n-mb** [comment]
Honestly, haven't heard about muon before yesterday.

Nonetheless, I added comments on issue #45 . Is it a case of incorrect docs in muon, that doesn't show *strong default* story, or is there an incorrect assumption about defaults here?

**Comment by 3n-mb** [comment]
Has anyone tried to put signal code into electron? It looks that signal is a chrome app, more of an extension, unlike, say, current [Cryptocat](https://github.com/cryptocat/cryptocat/blob/master/src/app.js). Since muon says to be extension-friendlier, signal on muon sounds to be a logical development.
*Disclamer:* I have no direct experience with extensions, so, these are vague guesses.
